### DISCUSSION TITLE
Would you want the Durable Object to persist by default?

### DISCUSSION THREAD

**Discussion by kwhitley** [kwhitley/itty-durable#16] [Polls]
Assuming there is a way to define your own desired behavior, what would you prefer the zero-config behavior to be?  Persist by default or not?

**Comment by zegevlier** [comment]
As normal durable objects do not persist by default, I think it would be confusing if itty durable objects did. If they do automatically persist, that should be made extremely clear in the docs (as it also adds costs that might not be necessary for your application.)

**Reply by kwhitley** [reply]
Very good point!

Per a conversation earlier, I'm also thinking I should expose the persist GET/PUT methods to allow overrides (for instance, if you wanted to persist somewhere other than the built-in DO storage).
